README.md

```diff
@@ -12,12 +12,17 @@ This project provides the core runtime that is used by the Wildfly application s
 Building
 -------------------
 
-Ensure you have JDK 8 (or newer) and Maven 3.2.2 installed, then run:
+Ensure you have JDK 8 (or newer) and run:
```

Comment:
-Ensure you have JDK 8 (or newer) and optionally Maven 3.3.3 (or newer) installed, then run: